How to generate Google API keys.md

```diff
@@ -23,18 +25,19 @@ Here's how to get its 3 access keys: `clientId`, `clientSecret`, `refreshToken`
 0. Visit https://console.developers.google.com/apis/credentials
 0. Click **Create credentials** > **OAuth client ID**:
 
-	<img width="771" alt="Create credentials" src="https://user-images.githubusercontent.com/1402241/77865679-e89f3a00-722f-11ea-942d-5245091f22b8.png">
+	> <img width="771" alt="Create credentials" src="https://user-images.githubusercontent.com/1402241/77865679-e89f3a00-722f-11ea-942d-5245091f22b8.png">
 
-0. Select **Other** (or **Desktop app** if available), enter `chrome-webstore-upload` and click **Create** 
+0. Select **Chrome App**, enter `Chrome Webstore Upload`, your extension’s ID, and click **Create** 
 
-	> <img width="187" alt="Configure client type" src="https://cloud.githubusercontent.com/assets/1402241/21517952/d1f36fce-cc97-11e6-92c0-de4485d97736.png">
+	> <img width="547" alt="Create OAuth client ID" src="https://user-images.githubusercontent.com/1402241/106205904-de6a0700-6184-11eb-8591-984e69c5e82a.png">
 
-0. Save your ✅ `clientId` and ✅ `clientSecret`, these are your 2 of your 3 keys.
+0. Save your ✅ `clientId` and ignore the `clientSecret`; `clientId` is 1 of 2 keys you will need.
 0. Place your `clientId` in this URL and open it:
 
-	`https://accounts.google.com/o/oauth2/auth?client_id=YOUR_CLIENT_ID&response_type=code&scope=https://www.googleapis.com/auth/chromewebstore&redirect_uri=urn:ietf:wg:oauth:2.0:oob`
+	`https://accounts.google.com/o/oauth2/auth?client_id=YOUR_CLIENT_ID&response_type=code&scope=https://www.googleapis.com/auth/chromewebstore&redirect_uri=urn:ietf:wg:oauth:2.0:oob&access_type=offline&approval_prompt=force`
```

Comment:
> The API in this case _does not_ return `refresh_token`, but just the `access_token`.

Maybe this was my fault, I did not use this updated URL. Now I used it and I got the `refresh_token` as expected 🥳 

<img width="246" alt="Screen Shot 8" src="https://user-images.githubusercontent.com/1402241/106214264-10369a00-6194-11eb-84fc-f124705a84c2.png">

Reply:
So....everything is good?

Reply:
The second problem seems fine, I just need to wait a little longer to ensure it doesn’t expire.

But it’s still a breaking change, so a few more changes are required as suggested.